modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -97,6 +114,43 @@ private Map<String, String> _getMap(String fileName) throws Exception {
 		return map;
 	}
 
+	private String _replaceImpl(
+		String content, String className, String newClassName) {
+
+		String[] lines = content.split(StringPool.NEW_LINE);
```

Comment:
Newlines can be represented differently on different platforms and operating systems (e.g. LF, CR, CRLF, etc.). So, in the future, use something like `StringUtil.splitLines()` when handling file contents.